src/components/UI/Typography/typography.story.tsx

```diff
@@ -1,5 +1,8 @@
 import type { Meta, StoryObj } from '@storybook/react';
 import { Typography } from './Typography';
+// ADD THESE IMPORTS FOR TESTING
```

Comment:
Those are comment generated by AI, please remove them. 
Its totaly normal to use AI to generate code but please remove the comments thank you